content/docs/for-buildpack-authors/how-to/write-buildpacks/add-labels.md

```diff
@@ -5,6 +5,56 @@ weight=99
 
 <!--more-->
 
-This page is a stub! The CNB project is applying to [Google Season of Docs](https://developers.google.com/season-of-docs/docs/timeline) to receive support for improving our documentation. Please check back soon.
+Labels are key-value pairs, stored as strings, that are attached to an image (i.e., arbitrary metadata). Labels are used to add helpful descriptions or attributes to an application image, which are meaningful to users.
 
-If you are familiar with this content and would like to make a contribution, please feel free to open a PR :)
+Labels are usually added at the time an image is created and can also be modified at a later time. Images can have multiple labels; however, each key must be unique.
```

Comment:
```suggestion
Labels are usually added at the time an image is created. Images can have multiple labels; however, each key must be unique.
```

In the buildpacks world, a buildpack writes labels to the `launch.toml` for the set of layers that it creates.  This is specified in https://github.com/buildpacks/spec/blob/main/buildpack.md#launchtoml-toml.  The contents of the various `launch.toml`s are assembled by `pack` (or `kpack` i.e. your _platform_) into the final output OCI Image.

Reply:
updated